lldb/source/Plugins/ObjectFile/Minidump/ObjectFileMinidump.cpp

```diff
@@ -55,6 +55,21 @@ size_t ObjectFileMinidump::GetModuleSpecifications(
   return 0;
 }
 
+struct SaveCoreRequest {
+  SaveCoreRequest(MinidumpFileBuilder &builder) : m_builder(builder) {}
+
+  ~SaveCoreRequest() {
+    if (!m_success)
+      m_builder.DeleteFile();
```

Comment:
We should emit visible error message telling end user that saving minidump has failed instead of silently fail and remove minidump file.

Reply:
I'll have to get the patch, it's one of my SBSaveCore API changes, but we now print the error when Plugin Manager returns, we could print twice, one specific for minidump